scripts/west_commands/runners/core.py

```diff
@@ -473,7 +473,7 @@ def create(cls, cfg: RunnerConfig,
           specified by ``add_parser()``.'''
         caps = cls.capabilities()
         if args.dev_id and not caps.dev_id:
-            _missing_cap(cls, '--dev-id')
+            _missing_cap(cls, '-i/--dev-id')
```

Comment:
Rebase cruft? Please remove this change.

Reply:
> Please remove this change.

Sorry for being vague. I meant "please make this '--dev-id', not '-i/--dev-id, for consistency with the rest of the `_missing_cap` calls". The previous version passed '--dev-id' to `_missing_cap` in one commit, then changed the string to '-i/--dev-id' in a subsequent commit. The first commit was correct.

Reply:
Oh, I see. I added it because the other ones did not have a short version of the command-line option and wanted the message for the user to be explicit about which option(s) were not supported. But not a big deal, just fixed it now.